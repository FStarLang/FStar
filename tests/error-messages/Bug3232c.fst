module Bug3232c

inline_for_extraction noextract
val f (x:int) : unit

[@@expect_failure]
noextract
let f x = ()
