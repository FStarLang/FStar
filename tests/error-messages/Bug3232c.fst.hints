[ "4e—\u0010\u001ee(ø*¡4\u0017*V4\"", [] ]