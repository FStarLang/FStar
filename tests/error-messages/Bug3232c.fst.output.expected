* Warning 240 at Bug3232c.fst(4,4-4,5):
  - Bug3232c.f is declared but no definition was found
  - Add an 'assume' if this is intentional

>> Got issues: [
* Error 93 at Bug3232c.fst(8,0-8,12):
  - Inconsistent qualifier annotations on Bug3232c.f
  - Expected '[inline_for_extraction, noextract]' got '[noextract]'
  - Only in declaration: '[inline_for_extraction]'

>>]
* Warning 240 at Bug3232c.fst(8,0-8,12):
  - Missing definitions in module Bug3232c: f

